src/.nuget/Microsoft.NETCore.Native/Microsoft.NETCore.Native.builds

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Project Sdk="Microsoft.Build.Traversal">
+<Project>
   <Import Project="Microsoft.NETCore.Native.props" />
-
-  <Import Project="$([MSBuild]::GetPathOfFileAbove(dir.props))" />
+  <Import Project="Sdk.props" Sdk="Microsoft.Build.Traversal" />
```

Comment:
What is the reason for importing the props and targets manually instead of letting the project do the import? Do we have conflicting imports?

If so can you comment them here?

Reply:
Directory.Build.props depends on the properties set in Microsoft.NETCore.Native.props. I've tried some build mysticism to have the Directory.Build.props pull in this props file if its needed. We'll see if it works.

Reply:
I would prefer setting the appropriate properties that drive the `Traversal` logic so they can be included naturally if possible.

Reply:
Couldn't you just rename `Microsoft.NETCore.Native.props` to `Directory.Build.props` and have it set the required properties before chaining in the `Directory.Build.props` from the parent folder?

Reply:
Hmm, that would probably work. I was under the impression that Directory.Build.props was always looked for in the directory above the project, but it seems reasonable that it might support in the same directory logic.